frontend/src/components/TabbedResults.less

```diff
@@ -86,9 +86,16 @@
         flex: 0 0 auto;
         display: flex;
 
-        li {
+        li:not(.history-tab-title) {
             flex: 0 1 auto;
             overflow: hidden;
+
+            > div {
```

Comment:
could you please use class name here?  It's hard to follow for which element you set this styles.

Reply:
This div is put by the bootstrap internals without any class name, I'd rather not customize it any more.

Reply:
ah. I see now why you do it.